lib/Sema/TypeCheckDecl.cpp

```diff
@@ -6061,6 +6061,20 @@ class DeclChecker : public DeclVisitor<DeclChecker> {
           new (TC.Context) OverrideAttr(SourceLoc()));
     }
 
+    // If the overridden method is declared in a Swift Class Declaration,
+    // dispatch will use table dispatch. If the override is in an extension
+    // warn, since it is not added to the class vtable.
+    //
+    // FIXME: Only warn if the extension is in another module, and if
+    // it is in the same module, update the vtable.
+    if (auto *baseDecl = dyn_cast<ClassDecl>(base->getDeclContext())) {
+      if (baseDecl->hasKnownSwiftImplementation() && 
+          !base->isDynamic() &&
+          override->getDeclContext()->isExtensionContext()) {
```

Comment:
Ahh thanks, I did not consider compatibility. I should be able to get a fix up soon.